forc/src/utils/defaults.rs

```diff
@@ -35,10 +35,10 @@ license = "Apache-2.0"
 
 [dependencies]
 fuels-abigen-macro = "0.1"
```

Comment:
I guess there was no change in this particular crate since `0.1`, but can you make it in line with the other `fuels-` crates?
```suggestion
fuels-abigen-macro = "0.2"
```